plover/assets/main.json

```diff
@@ -34977,6 +34986,8 @@
 "KEUPB/SHA/SA": "Kinshasa",
 "KEUPB/SHAU/SA": "Kinshasa",
 "KEUPB/SHEUP": "kinship",
+"KEUPB/STET/EUBG": "kinesthetic",
```

Comment:
Not pleading for or against this stoke, but a few examples to answer your question:
```
"STAT/EUS/STEUGS": "statistician",
"STAT/ST-BG": "statistic",
"SUS/STAEUP": "sustain",
```